tests/algorithms/lstm_classifier/lstm_classifier_test.py

```diff
@@ -0,0 +1,66 @@
+import unittest
+import sys
+import os
+from unittest.mock import MagicMock
+
+sys.path.append(os.path.realpath("."))
+
+import src.algorithms.lstm_classifier.lstm_classifier as lstm_classifier
+
+
+class LstmClassifierTest(unittest.TestCase):
+    def test_get_articles_and_labels(self):
+        review = [
+            {
+                "message": "I just heard about this budgeting app and I gave it a try and done.",
+                "derived-insight": {"category": "Application"},
+            }
+        ]
+        result = lstm_classifier.get_articles_and_labels(review)
+
+        # Check if stop words have been removed from the review message
+        self.assertFalse("and" in result[0][0])
+
+        # Check if review category is the same as label returned
+        self.assertEqual(result[1][0], "Application")
+
+    def test_split_data(self):
+        result = lstm_classifier.split_data(
+            ["Article Train-1", "Train-2", "Train-3", "Train-4", "Validation-5"],
+            [
+                "Label-Train-1",
+                "Label-Train-2",
+                "Label-Train-3",
+                "Label-Train-4",
+                "Label-Validation-5",
+            ],
+        )
+
+        # Check if the first 4 articles are a part of training data (0.8)
+        self.assertEqual(len(result[0]), 4)
```

Comment:
The numbers `4` and `1` should not be hard-coded. There is a variable exposed in the file [lstm_classifier.py#L36] called `TRAINING_PORTION`. Use that. The point being, the tests should not fail if i go and change the ratio.

[lstm_classifier.py#L36]: https://github.com/intuit/fawkes/blob/master/src/algorithms/lstm_classifier/lstm_classifier.py#L36